client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,35 @@
+
+import './styles.css';
+import { useLocation } from "react-router-dom";
+
+export const Profile = () => {
+  const { state } = useLocation();
+  const { name, middleName, surname, secondSurname, email, birthday } = state;
```

Comment:
This approach is not persistent, so this is ok for now, you have to improve in a future PR